docs/uenv-prgenv-gnu.md

```diff
@@ -7,6 +7,7 @@ It provides the GCC compilers (gcc, g++ and gfortran), MPI (cray-mpich), Python,
 The following packages are provided:
   
 * `aws-ofi-nccl`
+* `boost`
 * `cmake`
 * `cray-mpich`
     * built with `cuda` support on systems with NVIDIA GPUs
```

Comment:
I didn't update the mc recipe yet. Do we try to keep them in sync?

Reply:
We try to keep them in sync - but it isn't a strict requirement.
They main issue is that the mc version is used on Eiger, and we don't get any feedback that they are being used there.